### ISSUE TITLE
Propose attribute "content" from one datafile to another based on attribute names

### ISSUE THREAD

**Issue by yvanlebras** [earnaud/MetaShARK-v2#38]
So if some attributes names of a datafile are the same that those on another datafile (from the same datapackage (at least for now)), the user of the fill in module can have suggestion of content (or automatic prefill and validation ?) from on datafile to another

**Comment by earnaud** [comment]
Not sure if this would be really a good idea, as these samely-named attributes can actually differ. Also, I do not know yet which tool to use to achieve this. I will see this last.

**Comment by yvanlebras** [comment]
yes, you're right. ITM when these samely-named attributes are from different datatables from the same dataset, it often describe the same primary variable (here, on the DREAL dataset, there is plenty of this, and this is the second time I found it on a dataset). So maybe this can be relevant... or not ;)

**Comment by yvanlebras** [comment]
To reconsider as there was commnt on that at e-envir 2021
